### PR TITLE
feat(bulkmetadata): add StaticMetadata component

### PR DESCRIPTION
**Pull request by johan-hage**

### CODE REVIEW

File: src/components/StaticMetadata/StaticMetadata.test.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+describe('<StaticMetadata />', () => {
+	beforeEach(() => renderWithTheme(<StaticMetadata list={metadataList} />));
+
+	test('Renders with metadata', () => {
+		metadataList.forEach((metadata) => {
+			const labelColumn = screen.getByText(metadata.label);
+			expect(labelColumn).toBeInTheDocument();
+
+			if (typeof metadata.value === 'string') {
```
Comment (johan-hage) [inline src/components/StaticMetadata/StaticMetadata.test.tsx:15]: As we only render string values for the POC I ommited the Tag render testcase. If there is a wise solution available to check in the test if metadata.value is a type of React.ReactNode andso test the existence of the Tag output, glad to hear :-)

Comment (fortunato) [reply src/components/StaticMetadata/StaticMetadata.test.tsx:15]: You can mock `<Tag />` and spy on the mock being called with the appropriate props. Arguably though you only have to mock the description list component and check for the props on that. Coverage on the internals of the description list exist in the component library already.

File: src/components/StaticMetadata/__stubs__/metadataList.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { DescriptionList } from '@amsterdam/bmi-component-library';
+import { Tag } from '@amsterdam/asc-ui';
+
+export const metadataList: DescriptionList = [
```
Comment (fortunato) [inline src/components/StaticMetadata/__stubs__/metadataList.tsx:5]: I'm confused about this not throwing an error :thinking: 
I believe the appropriate type would be `ComponentProps<typeof DescriptionList>['list']`

File: src/components/StaticMetadata/StaticMetadata.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs'
```
Comment (fortunato) [inline src/components/StaticMetadata/StaticMetadata.stories.mdx:1]: Please convert this to .tsx so that we can enjoy type safety in storybook as well.

**Review by fortunato — CHANGES_REQUESTED** [review]
Looks good mostly :ok_hand:

File: src/components/StaticMetadata/StaticMetadata.test.tsx (hunk 1 above)
Comment (johan-hage) [reply src/components/StaticMetadata/StaticMetadata.test.tsx]: Changed.

**Review by johan-hage — COMMENTED** [review]
Updated by review comments.

File: src/components/StaticMetadata/StaticMetadata.test.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+import { metadataList } from './__stubs__';
+import { mockComponentProps, mocked } from '~/tests/helpers';
+import { DescriptionList } from '@amsterdam/bmi-component-library';
+
+jest.mock('@amsterdam/bmi-component-library', () => ({
+	...jest.requireActual('@amsterdam/bmi-component-library'),
+	DescriptionList: jest.fn().mockImplementation(() => <div data-testid="description-list" />),
+}));
+
+describe('<StaticMetadata />', () => {
```
Comment (fortunato) [inline src/components/StaticMetadata/StaticMetadata.test.tsx:13]: Neat :ok_hand:

**Review by fortunato — APPROVED** [review]
:clap: nicely done

**Pull request merged**